otter/integration/tests/test_lbsh.py

```diff
@@ -97,3 +98,82 @@ def test_oob_deleted_clb_node(self):
             ),
             timeout=timeout_default
         )
+
+    @tag("LBSH")
+    @inlineCallbacks
+    def test_move_node_to_oob_lb(self):
+        """
+        1 group, LB1 in config, LB2 not in any autoscale configs:
+
+        Server node moved from LB1 to LB2
+        Assert: Server put back on LB1
+        Assert: Server removed from LB2
+        """
+        # Create another loadbalancer not to be used in autoscale
+        # The CLB will not be added to the helper, since when the helper
+        # creates a group, it automatically adds the clb
+        clb_other = CloudLoadBalancer(pool=self.helper.pool)
+        clb_other.start(self.rcs, self)
```

Comment:
This seems to be called twice - once with yield, and here without.  Probably this line can be removed?